### DISCUSSION TITLE
Patreon Web Scraper & RSS Formatter

### DISCUSSION THREAD

**Discussion by mr-tchoo** [martinrotter/rssguard#957] [General]
Hi @martinrotter, 

I've created a small script to scrape posts from [Patreon](https://www.patreon.com) creators using the automation tool [Puppeteer](https://github.com/puppeteer/puppeteer).
 The repository also has information on how to use it with RSS Guard.

You can find it [here](https://github.com/mr-tchoo/patreon-2-rss).

Maybe you can add it to [this](https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#websites-scraping-) part of the documentation.

Thanks for working on this project and have a nice day. :grinning:

**Comment by martinrotter** [comment]
Hi. Wow what a nice script. Will add to docs later, crazily busy right now.

Btw, you can call "node" directly from RSS Guard, instead of calling to by some other way outside of RSS Guard (cron etc.).

**Comment by mr-tchoo** [comment]
Thanks, no problem, take your time. 👍 

And right I didn't think about the option to set a custom time on the feed which wouldn't get me rate limited directly in RSS Guard with the previous Patreon script, so I ran the script on a different schedule outside of it. Though rate limiting isn't as big of an issue with the new one. I will change the README. Good to know! 💡

**Comment by mr-tchoo** [comment]
I've added the option of dumping the feed as stdout and added a note on how and who might use it this way with RSS Guard.
